Update Example picture

Narrat opened the pull request:
Follow-up on https://github.com/soypat/sdf3ui/commit/f4e21d07d886e15c2885b5ce565a611ae2f87b48

soypat commented:
Snap- the curvature of the PolyMin function is not very visible. I hate to ask of more work from you but if you are not too busy is it possible to find values for the PolyMin so that the curvature is more visible like in the original picture?

Narrat commented:
It is really minimal :D
But sure, no problem asking for that. I will shortly force-push my branch with an adjusted example and a new picture.

Narrat committed:
Readme: update example picture

Plus a small update for the example, so it produces a more visible chamfer and looks more like the previous picture.

Narrat commented:
Please check the updated commit
